findbugs-exclude.xml

```diff
@@ -1,4 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2014-2019 Google LLC
```

Comment:
The year for the copyright should be the year the file was first published.
s/2014-2019/2014
Here and throughout the PR.

Reply:
Done.